external-table/src/gpdbwritableformatter.c

```
@@ -663,6 +669,23 @@ gpdbwritableformatter_export(PG_FUNCTION_ARGS)
 	PG_RETURN_BYTEA_P(myData->export_format_tuple->data);
 }
 
+static void
+parse_params(FunctionCallInfo fcinfo, format_t *myData) {
```

Should we be strict here in parsing extra options to the formatter? If we encounter anything other than `dump_core_on_error`, should we report an error? This might help catch user typos (e.g., `dump_core_on_eror`).